.env

```diff
@@ -18,6 +18,8 @@ HASH_PG_PASSWORD=postgres
 HASH_PG_PORT=5432
 HASH_PG_USER=postgres
 
+HASH_MIGRATION_PG_URL=postgres://postgres:postgres@localhost/postgres
```

Comment:
This was the quickest way for me to get this up and running. I've had a brief discussion w/ @kachkaev that we would ideally define the scripts that run our migrations in a collection of `scripts/*.ts` files, so that we can better control the importing/passing of environment variables (by importing `packages/hash/backend-utils/src/environment.ts` for instance). This would likely involve calling `node-pg-migrate` using `execa`.

To reduce the scope of this PR, and get it done quicker I've managed to get migration scripts working without these wrapper scripts, by interfacing with `node-pg-migrate` directly in the `datastore/package.json` using the `pg:migrate` shorthand. I've created [a follow-up task](https://app.asana.com/0/0/1202032662538879/f) to capture this, but I'm not sure about the priority. This should probably be done when we run into some kind of limitation with the existing migration configuration, where a custom script would solve our issue.